OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -40,6 +40,9 @@ public class ProductionQueueInfo : ITraitInfo
 		[Desc("The build time is multiplied with this value on low power.")]
 		public readonly int LowPowerSlowdown = 3;
 
+		[Desc("Whether this queue tries to build in parallel.")]
+		public readonly bool ParallelBuild = false;
```

Comment:
Can you please split this out into a `ParallelBuildProductionQueue`, like we do for `ClassicProductionQueue`?